fixtures/arithmeticops.cpp.sem.uast

```diff
@@ -28,1429 +27,44 @@
-                     '@token': "%",
-                     '@role': [Arithmetic, Binary, Expression, Modulo],
+               Node: { '@type': "uast:Function",
+                  Body: { '@type': "uast:Block",
                      '@pos': { '@type': "uast:Positions",
                         start: { '@type': "uast:Position",
-                           offset: 43,
-                           line: 6,
-                           col: 3,
+                           offset: 11,
+                           line: 1,
+                           col: 12,
                         },
                         end: { '@type': "uast:Position",
-                           offset: 46,
-                           line: 6,
-                           col: 6,
+                           offset: 142,
+                           line: 19,
+                           col: 2,
                         },
                      },
-                     ExpressionType: "int",
-                     ExpressionValueCategory: "PRVALUE",
-                     IsLValue: false,
-                     'Prop_Operand1': { '@type': "cpp:CPPASTLiteralExpression",
-                        '@token': "1",
-                        '@role': [Binary, Expression, Left, Literal, Number],
-                        '@pos': { '@type': "uast:Positions",
-                           start: { '@type': "uast:Position",
-                              offset: 43,
-                              line: 6,
-                              col: 3,
-                           },
-                           end: { '@type': "uast:Position",
-                              offset: 44,
-                              line: 6,
-                              col: 4,
-                           },
-                        },
-                        ExpressionType: "int",
-                        ExpressionValueCategory: "PRVALUE",
-                        IsLValue: false,
-                        kind: "integer_constant",
-                     },
-                     'Prop_Operand2': { '@type': "cpp:CPPASTLiteralExpression",
-                        '@token': "2",
-                        '@role': [Binary, Expression, Literal, Number, Right],
-                        '@pos': { '@type': "uast:Positions",
-                           start: { '@type': "uast:Position",
-                              offset: 45,
-                              line: 6,
-                              col: 5,
-                           },
-                           end: { '@type': "uast:Position",
-                              offset: 46,
-                              line: 6,
-                              col: 6,
-                           },
-                        },
-                        ExpressionType: "int",
-                        ExpressionValueCategory: "PRVALUE",
-                        IsLValue: false,
-                        kind: "integer_constant",
-                     },
+                     Statements: [],
```

Comment:
It looks like the whole function body was discarded. Please diff `.uast` vs `.sem.uast` before approving it as a correct fixture test file.

Reply:
Yes, I'm investigating it, I was so focused on getting the parameters right that I didn't check the body :/.